p2p/src/main/java/bisq/network/p2p/network/Connection.java

```diff
@@ -393,7 +432,13 @@ private boolean violatesThrottleLimit(NetworkEnvelope networkEnvelope) {
     @Override
     public void onMessage(NetworkEnvelope networkEnvelope, Connection connection) {
         checkArgument(connection.equals(this));
-        UserThread.execute(() -> messageListeners.forEach(e -> e.onMessage(networkEnvelope, connection)));
+
+        if(networkEnvelope instanceof BundleOfEnvelopes)
```

Comment:
Style is off (missing space). Best to use some tool that handles that automatically.

Reply:
ahm. That is what Intellij does (is that the tool you are referring to?). Besides, I do not spot the issue